common/network-common/src/test/java/org/apache/spark/network/client/TransportClientFactorySuite.java

```diff
@@ -224,4 +226,23 @@ public void closeFactoryBeforeCreateClient() throws IOException, InterruptedExce
     factory.close();
     factory.createClient(TestUtils.getLocalHost(), server1.getPort());
   }
+
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+
+  @Test
+  public void fastFailConnectionInTimeWindow() throws IOException, InterruptedException {
+    TransportClientFactory factory = context.createClientFactory();
+    TransportServer server = context.createServer();
+    server.close();
+    int unreachablePort = server.getPort();
```

Comment:
seems safer to get port before close() just incase the close call cleans it up or makes it inaccessible.

Reply:
I think It is not necessary.
I have checked the code, the port would be only set value during init()

Reply:
have modified the code, I think it would not affect the result of ci.

Reply:
thanks, I understand it works now, its just future proofing it incase someone decided to clear port on close or something.

Reply:
got